bin/mocha.js

```diff
@@ -109,7 +110,12 @@ if (mochaArgs['node-option'] || Object.keys(nodeArgs).length || hasInspect) {
   proc.on('exit', (code, signal) => {
     process.on('exit', () => {
       if (signal) {
-        process.kill(process.pid, signal);
+        if (mochaArgs['posix-exit-codes'] === true) {
+          process.exitCode = 128 + os.constants.signals[signal];
+          process.exit(process.exitCode);
```

Comment:
[Question] From https://nodejs.org/api/process.html#processexitcode_1:

> Specifying a code to [`process.exit(code)`](https://nodejs.org/api/process.html#processexitcode) will override any previous setting of process.exitCode.

Is there a reason to set both?